docs/surveys/salary-survey/2022.rst

```diff
@@ -14,7 +14,2069 @@ Write the Docs Salary Survey 2021 Results
+   +--------------------------+-----+---------------------+
+   | 1,001-10,000 employees   | 126 |             $79,989 |
+   +--------------------------+-----+---------------------+
+   | 10,001-100,000 employees |  50 |             $77,235 |
+   +--------------------------+-----+---------------------+
+   | 100,001+ employees       |  40 |             $95,127 |
+   +--------------------------+-----+---------------------+
+
+Benefits
+--------
+
+In almost all countries apart from the US, employees are entitled to paid vacation time and paid sick leave by law, and many also mandate pension contributions and/or paid parental leave. Similarly, many countries have some form of universal health care, negating the need for employer-provided health cover. To make this clearer, we asked respondents to only check the boxes for vacation time, health insurance, pension plans and parental leave if their employee benefit was in excess of what is required by law in the country where they live.
+
+.. container:: question
+   
+   .. rubric:: What we asked
+
+   Does your salary package include any additional benefits? 
+
+   - Paid vacation time (in excess of government-mandated minimums)
+   - Paid parental leave (in excess of government-mandated minimum)
+   - Time off or bonuses for community-related activities
+   - Unlimited PTO (paid/personal time off)
+   - Health insurance (in excess of government-mandated minimums)
+   - Other types of insurance e.g. life insurance, accident insurance, income protection insurance
+   - Pension, superannuation, or retirement fund (in excess of any government-mandated minimums)
+   - Stocks, shares, stock options, or equity
+   - Commission or bonus payments
+   - Professional development / ongoing education / conference budget
+   - Meals, meal vouchers, or food-related benefits
+   - Gym, fitness, sport, or other wellness-related benefits
+   - Transportation-related benefits (company car, public transport passes, parking, fuel vouchers or reimbursements for any transport-related cost)
+   - Home office or co-working office budget (including for laptops or other equipment)
+   - Phone and/or internet-related benefits or reimbursements
+   - None of the above
+   - Other (please specify)
+
+A small number of respondents (4.1, representing 20 individuals) indicated that they did not receive any of the listed benefits.
+
+.. table:: Benefits (2022)
+   :width: 100%
+   :name: tbl-2022-employee-benefits
+
+   +-------------------------------------------------------------------------------------------------------------------------------------------------+-----+-------+
+   | Benefit                                                                                                                                         | No. | %     |
+   +=================================================================================================================================================+=====+=======+
```

Comment:
What are the asterisks?

Reply:
Good catch, they're ones that are "above any goverment-mandated minimums", I will add that note.